inRange.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Checks if the given number is between the given start and end numbers. Including the start number but excluding the end.
+ * @param {number} numberToCheck - The number to be checked
+ * @param {number} startNumber - The start of the range, defaulted at 0
+ * @param {number} endNumber - The end of the range
+ * @returns {boolean}
+ */
+
+export default function inRange(
+  numberToCheck,
+  startNumber = 0,
+  endNumber) {
+
+  let start = startNumber
+  let end = endNumber
+
+  if (startNumber > endNumber || endNumber === undefined) {
+    start = endNumber
+    end = startNumber
+    startNumber = start
+    endNumber = end
+  }
+
+  if (!Number(startNumber)) {
```

Comment:
shouldn't this be the first check you do?

Reply:
No, because then the startnumber will be undefined after the second ifstatement and not turn into a 0. Also, two others tests (all in all 3 tests) will not pass if that is the first check I do.

Reply:
I would rather see you use `Number.isNaN(Number(...))` here